Rubberduck.Core/AddRemoveReferences/IReferenceInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rubberduck.AddRemoveReferences
+{
+    public interface IReferenceInfo
```

Comment:
It is a bit confusing to have a new interface with this name in `Core` when we have a readonly struct `ReferenceInfo` with nearly the same members in `VBEditor`.

Reply:
Agreed. I'm going to need another implementation to save recent and pinned items in the General settings too. I'll sort out an interface hierarchy when I get that wired up.